src/client/app/containers/CompareChartContainer.ts

```diff
@@ -4,21 +4,35 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { Bar } from 'react-chartjs-2';
-import moment from 'moment';
+import { Bar, ChartComponentProps } from 'react-chartjs-2';
+import { ChartData, ChartDataSets } from 'chart.js';
+import * as moment from 'moment';
 import { connect } from 'react-redux';
-import datalabels from 'chartjs-plugin-datalabels'; // eslint-disable-line no-unused-vars
+// This is better than using an import, since we don't actually use anything from the plugin in the code.
+/// <reference path="chartjs-plugin-datalabels" />
```

Comment:
What is the purpose of this line?

Reply:
Chart.js plugins need to be _imported_ to work, but will never be used directly. Therefore it's better to tell the TypeScript compiler to just pull it in than to use an import and incur a warning about lack of types (which will never be fixed, I contacted the developer).